packages/ember-metal/lib/property_get.js

```diff
@@ -76,7 +76,7 @@ export function _getPath(root, path) {
 
   for (let i = 0; i < parts.length; i++) {
     if (obj == null) {
```

Comment:
Falsey alone won't work because of empty strings, e.g.

`Ember.get({ foo: "" }, 'foo.length')` should be 0 not undefined. (Can you add a test for this one @robbiepitts?)